flight/Modules/ManualControl/tablet_control.c

```diff
@@ -97,8 +97,9 @@ int32_t tablet_control_select(bool reset_controller)
 
 			// Command to not move. This code is identical to that in manualcontrol
 			LoiterCommandData loiterCommand;
-			loiterCommand.Forward = 0;
-			loiterCommand.Right = 0;
+			loiterCommand.Pitch = 0;
+			loiterCommand.Roll = 0;
+			loiterCommand.Throttle = 0.5f;
```

Comment:
Could you make this something really low then like 0.1 and then update it in the subsequent PR. I'd hate this to get forgotten and one day cause an unexpected climb. Sorry for being pedantic, but flyaway are my biggest fear.